packages/render/src/shared/useRenderingOptions.tsx

```
@@ -0,0 +1,11 @@
+import { Options } from "./options";
+
+const opts: Record<string, Options> = {}
+
+export function setRenderingOptions(uniqueRenderId: string | undefined, options: Options | undefined) {
+  opts[uniqueRenderId ?? ""] = options ?? ({} as Options);
```

Could you throw an error here if the options for this ID are already defined?

---

Yes, added